app/src/app/[lng]/data/[step]/page.tsx

```diff
@@ -808,67 +822,69 @@ export default function AddDataSteps({
               </Box>
               <Box display="flex" flexDirection="column" gap="8px">
                 {sectorData &&
-                  sectorData[0]?.files.map((file: any, i: number) => (
-                    <Card
-                      shadow="none"
-                      h="80px"
-                      w="full"
-                      borderWidth="1px"
-                      borderColor="border.overlay"
-                      borderRadius="8px"
-                      px="16px"
-                      py="16px"
-                      key={i}
-                    >
-                      <Box display="flex" gap="16px">
-                        <Box>
-                          <ExcelFileIcon />
-                        </Box>
-                        <Box
-                          display="flex"
-                          flexDirection="column"
-                          justifyContent="center"
-                          gap="8px"
-                        >
-                          <Heading
-                            fontSize="lable.lg"
-                            fontWeight="normal"
-                            letterSpacing="wide"
-                            isTruncated
-                          >
-                            {file.fileName}
-                          </Heading>
-                          <Text
-                            fontSize="body.md"
-                            fontWeight="normal"
-                            color="interactive.control"
+                  sectorData[0]?.files.map((file: any, i: number) => {
+                    return (
+                      <Card
+                        shadow="none"
+                        h="84px"
+                        w="full"
+                        borderWidth="1px"
+                        borderColor="border.overlay"
+                        borderRadius="8px"
+                        px="16px"
+                        py="16px"
+                        key={i}
+                      >
+                        <Box display="flex" gap="16px">
+                          <Box>
+                            <ExcelFileIcon />
```

Comment:
This icon should not be hardcoded but dynamic based on the file's MIME type or file extension.